<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Principale</class>
 <widget class="QMainWindow" name="F_Principale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbOperationEnCours">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>8</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pas d'opération en cours</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLAS">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de l'operation:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>20</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LAS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Articles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbArticle1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>111</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnInitialiserOperation">
       <property name="text">
        <string>Initialiser opération</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAjouterArticle">
       <property name="text">
        <string>Ajouter article</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAjouterLot">
       <property name="text">
        <string>Ajouter Lot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuitterOperation">
       <property name="text">
        <string>Quitter opération</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbArticle2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbArticle3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Lot:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLot1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLot2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLot3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contenants:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbContenant1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbContenant2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbContenant3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QTreeView" name="tvLAS">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>lbOperationEnCours</zorder>
   <zorder>infoLAS</zorder>
   <zorder>label</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lbArticle1</zorder>
   <zorder>lbArticle2</zorder>
   <zorder>lbArticle3</zorder>
   <zorder>label_4</zorder>
   <zorder>lbLot1</zorder>
   <zorder>lbLot2</zorder>
   <zorder>lbLot3</zorder>
   <zorder>label_5</zorder>
   <zorder>lbContenant1</zorder>
   <zorder>lbContenant2</zorder>
   <zorder>lbContenant3</zorder>
   <zorder>tvLAS</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
